backend/src/parser/TheParser.ts

```diff
@@ -683,6 +687,17 @@ export default class TheParser {
     return { ...result, text }
   }
 
+  parseGallery(node: Element): ParseResult {
+    const showArrows = node.attribs['show-arrows'] !== 'false'
```

Comment:
let's prefix those custom attributes with `data-` to make them standard-compliant